cmd/cli/verify_connectivity.go

```diff
@@ -0,0 +1,25 @@
+package main
+
+import (
+	"io"
+
+	"github.com/spf13/cobra"
+	"helm.sh/helm/v3/pkg/action"
+)
+
+const verifyConnectivityDescription = `
+This command consists of multiple subcommands related to verifying
+connectivity related configurations.
+`
+
+func newVerifyConnectivityCmd(config *action.Configuration, stdout io.Writer, stderr io.Writer) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "connectivity",
+		Short: "verify connectivity configurations",
+		Long:  verifyConnectivityDescription,
+		Args:  cobra.NoArgs,
+	}
+	cmd.AddCommand(newVerifyConnectivityPodToPodCmd(config, stdout, stderr))
```

Comment:
Should we signify in some way that this isn't complete yet? Or is the plan to just get feedback and continue implementation prior to merge?

Reply:
Yes this PR is to just gather feedback on the idea. It cannot be merged in its current state hence still a draft PR.